### PR TITLE
Add context propagation example from tutorial

### PR DESCRIPTION
**Pull request by gastaldi**
**Check list**:

Your pull request:

- [x] targets the `development` branch
- [x] uses the `999-SNAPSHOT` version of Quarkus
- [ ] has tests (`mvn clean test`)
- [x] works in native (`mvn clean package -Pnative`)
- [ ] has native tests (`mvn clean verify -Pnative`)
- [x] makes sure the documentation must not be updated
- [ ] links the documentation update pull request (if needed)
- [ ] updates or creates the `README.md` file (with build and run instructions)
- [ ] For new quickstart, is located in the directory _component-quickstart_
- [ ] For new quickstart, is added to the root `pom.xml` and `README.md`

### CODE REVIEW

File: context-propagation/pom.xml [hunk 1]
```diff
@@ -54,6 +54,14 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jdbc-postgresql</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vertx-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.smallrye.reactive</groupId>
+            <artifactId>smallrye-axle-web-client</artifactId>
```
Comment (cescoffier) [inline context-propagation/pom.xml]: Axle being deprecated, it would be better to switch to Mutiny.

File: context-propagation/pom.xml [hunk 2]
```diff
@@ -54,6 +54,14 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jdbc-postgresql</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vertx-web</artifactId>
```
Comment (cescoffier) [inline context-propagation/pom.xml]: What's the reason ?

Comment (gastaldi) [reply context-propagation/pom.xml]: WebClient and `response.bodyAsJsonObject`

**Comment by gastaldi** [comment]
Closing this as it is no longer needed

**Pull request closed without merging**